export const displayNoResultMessage = setError => {
  setError(
    "Apologies, but it seems that something didn't go as planned. Please bear with us and try again later."
  );
};
